osu.Game.Tournament/Screens/Board/Components/ModChessCounterText.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Specialized;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterfaceFumo;
+using osu.Game.Tournament.Models;
+using osuTK.Graphics;
+
+namespace osu.Game.Tournament.Screens.Board.Components
+{
+    public partial class ModChessCounterText : TournamentSpriteText
+    {
+        public readonly string ModAcronym;
+
+        [Resolved]
+        private LadderInfo ladder { get; set; } = null!;
+
+        private int? currentCount;
+
+        public ModChessCounterText(string acronym)
+        {
+            ModAcronym = acronym;
+            var colourScheme = ModColours.FromModString(acronym);
+
+            Colour = colourScheme.Accent;
+            Font = OsuFont.Torus.With(weight: FontWeight.SemiBold, size: 18);
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            ladder.CurrentMatch.BindValueChanged(matchChanged);
+            ladder.CurrentMatch.Value?.Round.BindValueChanged(_ => updateCounter());
+            ladder.CurrentMatch.Value?.Round.Value?.Beatmaps.BindCollectionChanged((_, _) => updateCounter());
+
+            if (ladder.CurrentMatch.Value != null)
+                ladder.CurrentMatch.Value.ChessPlacements.CollectionChanged += placementChanged;
+
+            updateCounter();
+        }
```

Comment:
```suggestion
        protected override void LoadComplete()
        {
            base.LoadComplete();

            ladder.CurrentMatch.BindValueChanged(matchChanged, true);
        }
```
the logic here is repeat with the `matchChanged`
we can call matchChanged directly by set `runOnceImmediately`  as true

Reply:
In fact, I would prefer to maintain a bindable currentMatch locally